index.html

```diff
@@ -1755,6 +1854,96 @@ <h2>
         </ol>
       </section>
     </section>
+    <section>
+      <h2>
+        ProtocolHandler items
+      </h2>
+      <p>
+        Each <dfn>protocol handler description</dfn> is an [=object=] that represents
+        represents a protocol that the web application wants to handle. It has
+        the following members:
+      </p>
+      <ul>
+        <li>[=`protocol`=]
+        </li>
+        <li>[=`url`=]
+        </li>
+      </ul>
+      <p>
+        A user agent SHOULD use these values to register the web application as
+        a handler with the operating system. When the user activates a protocol
+        handler URL, the user agent SHOULD run <a>Handling a protocol
+        launch</a>.
+      </p>
+      <p class="note">
```

Comment:
Do we need to talk about `.registerProtocolHandler()` at all? The API is not really relevant here?

Reply:
The process itself is the same. This is the way to handle the protocol launch from the manifest instead of from the `registerProtocolHandler()` API. Furthermore, following this link explains that the "_same steps used to invoke a protocol handler defined in HTML_" are the ones that apply. These steps are the ones defined in [the `.registerProtocolHandler()` method steps](https://html.spec.whatwg.org/multipage/system-state.html#protocol-handler-invocation). Since this is another entry point to the steps and that it is stated that they are the same ones defined in the HTML spec, I don't think it impacts the mentioned API nor that it is relevant here.